### ISSUE TITLE
How to avoid log zero in EntropicConfusion ?

### ISSUE THREAD

**Issue by QtacierP** [abhimanyudubey/confusion#12]
Thanks for your great paper and related codes. However, I still met some problems in my own implementation with Keras. How to initialize weights better? I mean that if I use my default setting from your PyTorch codes, the feature map in EntropicConfusion hasn't been normalized, and the Log will get a NAN problem. I think I should add an extra normalization?
